docs/source/slurmguide/slurmmonitoringandmanaging.md

```diff
@@ -1,33 +1,20 @@
 (slurm-monitoring-and-managing)=
 # Monitoring and Managing Jobs
```

Comment:
I feel like the overall organization of this page needs some work. There are a lot of headings that have only one sentence to describe the section/sections below.

Reply:
It might be good to go through this page together.